src/quic/node_quic_session.cc

```diff
@@ -1367,35 +1367,35 @@ QuicSession::QuicSession(
 
   session_stats_.created_at = uv_hrtime();
 
-  if (wrap->DefineOwnProperty(
+  CHECK(wrap->DefineOwnProperty(
           env()->context(),
           env()->state_string(),
           state_.GetJSArray(),
-          PropertyAttribute::ReadOnly).IsNothing()) return;
+          PropertyAttribute::ReadOnly).IsJust());
```

Comment:
I prefer handling errors, but if you do this, there is a `.Check()` method on `Maybe<...>` that we tend to prefer in our codebase over `CHECK((...).IsJust());` (it does the same thing)

Reply:
Before it moves out of experimental, we'll handle these. For now while we're actively developing, I want to make them checks because they really shouldn't fail but I want to make sure while we're going through building tests. I'll add a todo note to handle them.

Reply:
(note, they weren't being properly handled previously)